.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,15 @@
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may delay the bug being addressed. Thanks!
+-->
+
+**Bug description**
+
+**Expected behavior**
+
+**Steps to reproduce the bug**
+
+**Version** (include the SHA if the version is not obvious)
+
+**Environment where the bug was observed (cloud, OS, etc)**
```

Comment:
add TAP version, K8s version - and ask team for any other relevant env info that would be helpful and add it.